src/app/modules/group/components/user-list/user-list.component.html

```diff
@@ -1,29 +1,23 @@
-<p-progressSpinner *ngIf="state === 'loading'"></p-progressSpinner>
-
 <p *ngIf="state === 'error'">Error while loading the group members</p>
 
-<p *ngIf="state === 'empty'">This group is empty, check below, the different ways to add users or sub-groups.</p>
-
 <alg-grid
-*ngIf="state === 'ready'"
-[data]="members"
-[showGear]="false"
+  [data]="members"
+  [showGear]="false"
+  (sort)="onCustomSort($event)"
+  [loading]="state === 'loading'"
```

Comment:
The idea was to keep the spinner the first time but this table loading when only the sort is changing. It is a bit cheating but maybe you could rely on `state === 'loading' && members.length > 0`

Reply:
fixed